tutorials/balance_quickstart_ipw.ipynb

```diff
@@ -0,0 +1,495 @@
+{

```

Comment:
Since 
tutorials/balance_quickstart_ipw.ipynb
is identical to
tutorials/balance_quickstart.ipynb, just with a few more examples - I suggest you just add them to 
tutorials/balance_quickstart.ipynb